app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryFragment.kt

```diff
@@ -61,7 +61,7 @@ import org.kiwix.kiwixmobile.zim_manager.library_view.adapter.LibraryListItem
 import org.kiwix.kiwixmobile.zim_manager.library_view.adapter.LibraryListItem.BookItem
 import javax.inject.Inject
 
-class LibraryFragment : BaseFragment() {
+open class LibraryFragment : BaseFragment() {
```

Comment:
this doesn't need to be open or really exist at all, should just be OnlineLibraryFragment